statusHandler/p2pQuota/p2pQuotaProcessor.go

```diff
@@ -0,0 +1,135 @@
+package p2pQuota
+
+import (
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/statusHandler"
+)
+
+type quota struct {
+	numReceivedMessages   uint32
+	sizeReceivedMessages  uint64
+	numProcessedMessages  uint32
+	sizeProcessedMessages uint64
+}
```

Comment:
why uint32 for the num* fields ? at 35000 transactions / second and one message per transaction you get 2^32/35000/86400 = 1.42 days until the number wraps around...

Reply:
the counter is supposed to reset in one second or so. Also, we are sending transactions in packets of 400 or so (aprox 128KB chunks)